layouts/footer.slim

```diff
@@ -5,5 +5,9 @@ footer.ui.center.aligned.container lang='en'
 script src="/static/dist/semantic-ui/semantic.min.js"
 script src="/static/javascripts/ga.js"
 script src="/static/javascripts/jquery.address.js"
+script src="/static/javascripts/cookie.js"
+script src="/static/javascripts/clipboard.js"
+script src="/static/javascripts/less.js"
```

Comment:
To answer your question, "- less.js: parse html, not quite clear about what it does", it seems that by loading a `less.js`, you can write `less` code directly in a web page.